jekyll/_cci2/monitoring.adoc

```diff
@@ -12,15 +12,64 @@ This section includes information on metrics for monitoring your CircleCI Server
 
 toc::[]
 
-== System Monitoring
+== Metrics Overview
 
-To enable metrics forwarding to either AWS Cloudwatch or Datadog, navigate to your CircleCI Management Console, select Settings from the menu bar and scroll down to enable the provider of your choice (your-circleci-hostname.com:8800/settings#cloudwatch_metrics).
+Metrics are technical statistical data collected for monitoring and analytics purposes. The data includes basic information, such as CPU or memory usage, as well as more advanced counters, such as number of executed builds and internal errors. Using metrics you can:
+
+* Quickly detect incidents and abnormal behavior
+* Dynamically scale compute resources
+* Retroactively understand infrastructure-wide issues
+
+=== How Metrics Work in CircleCI Server
+
+Telegraf is the main component used for metrics collection in CircleCI Server. Telegraf is server software that brokers metrics data emitted by CircleCI services to data monitoring platforms such as Datadog or AWS CloudWatch.
+
+.Metrics
+image::metrics.png[Metrics]
+
+Metrics collection in CircleCI Server works as follows:
+
+* Each component of a Server installation sends metrics data to the telegraf container running on the Services machine.
+* Telegraf listens on port 8125/UDP and receives data from all components (inputs) and applies configured filters to determine whether data should be kept or dropped.
+* For some metric-types, Telegraf keeps metrics data inside and calculates statistical data (such as max, min, mean, stdev, sum) periodically.
+* Finally, Telegraf sends out data to stdout and configured data platforms (outputs), such as stdout, Datadog and/or AWS CloudWatch.
```

Comment:
"stdout" appears twice (I realized that this is a fault in my original draft 😇 )

How about like this:
```
Finally, Telegraf sends out data to configured sinks (outputs), such as stdout on the Services box, Datadog and/or AWS CloudWatch.
```